helm/dashboards/charts/public_dashboards/dashboards/shared/public/apiserver.json

```
@@ -1618,8 +1618,8 @@
      ]
    },
    "timezone": "UTC",
-   "title": "Loreno api server test",
+   "title": "Kubernetes / API server",
    "uid": "09ec8aa1e996d6ffcd6817bbaff4db1basd",
```

You could want to have an easy-to-read UID.
Like:
```suggestion
   "uid": "k8s-apiserver",
```

This would be useful when adding a link to the dashboard from prometheus-rules/alerts.

---

Definitely, thanks!